.env

```diff
@@ -0,0 +1,5 @@
+FACTOMD_HOST=localhost
+FACTOMD_PORT=8088
+FACTOM_WALLETD_HOST=localhost
```

Comment:
I do see it referenced in the identityclient, but the implementation does nothing with walletd. Suggest to either mention that as a todo or simply remove it for now.

Reply:
Good point. Removed.